fields/acf-Typography-v4.php

```diff
@@ -484,6 +484,8 @@ function create_field( $field )
 				}
 				
 				if( in_array($f, $numbers) ){ ?>
+                
+                
```

Comment:
Empty line breaks do not make any sense here. I think majority of developers (including me) do not use empty line breaks between PHP and HTML LOC.